config/initializers/barong.rb

```diff
@@ -38,6 +38,7 @@
 
 Barong::App.define do |config|
   config.set(:app_name, 'Barong')
+  config.set(:session_auto_renew, false)
```

Comment:
value should be in strings
lets stick to 'enabled' and 'disabled'

also var name should be prefixed with 'barong'